src/architecture/configuration.md

```diff
@@ -42,14 +42,25 @@ This categorization holds both for core and service parameters.
     | Global  | Validators’ public keys | Anchoring address |
     | Local   | Validator’s private key | RPC params for Bitcoin Core |
 
+!!! tip
+    See [sample configuration file](https://github.com/exonum/exonum/blob/master/exonum/tests/testdata/config/config02.toml)
+    for reference.
+
 ### Global Parameters
 
 #### [genesis]
 
 The configuration used to create a genesis block.
 
-- **validators**  
-  List of validators’ public keys as hex strings
+#### [genesis.validator_keys]
+
+List of validators’ public keys as hex strings. Each list element consists of
+two parts:
+
+- **consensus_key**  
+  Validator's public key (hex) for dealing with consensus messages.
```

Comment:
Please use smart apostrophes `’`, like in the rest of the article.

Also, I think "...for use with consensus messages" sounds a bit more natural.